core/src/apps/cardano/sign_tx/pool_owner_signer.py

```diff
@@ -27,23 +27,25 @@ def __init__(
     ) -> None:
         super().__init__(ctx, msg, keychain)
 
+        self.signing_mode_title = "Confirming a transaction as pool owner."
```

Comment:
This string seems to never be used, I find it a bit confusing (maybe at least a comment with explanation?).

Reply:
It is now used since the init dialog can be shown if pool registration tx contains auxiliary data (https://github.com/vacuumlabs/trezor-firmware/pull/79/commits/eddc71d2c1978a113e8acefe7ac1f7c77d413067).